### DISCUSSION TITLE
How do i restart an already existing server with the docker image

### DISCUSSION THREAD

**Discussion by TDPFerox** [itzg/docker-minecraft-server#1886] [Q&A] [answered]
Hello 

Im a new user to linux and docker. I just started using these a few days ago.
Im running an ubuntu server 22.04 lts with docker installed. 
At the moment im trying to migrate my already running Forge Minecraft server into a docker container.

The Minecraft Server is a Forge 1.19.2 ATM8 Server running the 1.0.6.b zip from curseforge
Forge Version: 43.1.55

I was already able to download the image and get a normal forge server to run without any mods (was also able to connect to it)

now the next step was to migrate my world directory into the data path aswell as the mods and the config files because i dont wanted to have to go through editing all configs again

i already managed to start the server as a container once but after closing it down and trying to restart it again i just get error messages

the command i used was:

`docker run -it -v /home/mc:/data \
    -e TYPE=FORGE \
    -e VERSION=1.19.2 -e FORGE_VERSION=43.1.55 \
    -e MAX_MEMORY=14G \
    -p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server`

the result was:

![Docker Fehler](https://user-images.githubusercontent.com/105038214/210465683-98e1a5e8-ed17-41c7-9b1a-551cb5f37509.PNG)

**how do i avoid it trying to reinstall the server that is already there with all configs and stuff?**

here an image of my data directory:

![Docker Directory](https://user-images.githubusercontent.com/105038214/210465921-e65ac821-bc6f-410a-9295-49988fddf662.PNG)

**Comment by itzg** [comment, marked as answer]
It looks like you have run out of disk space

![image](https://user-images.githubusercontent.com/988985/210668641-e338dbf3-6195-4033-89a6-188c83cddd29.jpeg)

**Reply by TDPFerox** [reply]
So the command is ok as it is?

**Reply by itzg** [reply]
Yes, assuming you put everything from your previous setup in `/home/mc`

**Reply by TDPFerox** [reply]
Thank you so much for your help. Now it works thanks to you 🥇
